espree.js

```diff
@@ -5114,7 +5113,9 @@ function parseClassDeclaration() {
 
     expectKeyword("class");
 
-    id = parseVariableIdentifier();
+    if (lookahead.type === Token.Identifier) {
```

Comment:
Just FYI, the problem case is:

```js
class {}
```

This should throw an error because class declarations require names, but this change parsed it okay.